### PR TITLE
small bugs (#14 #15)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
#14 
#15 
Taipy-studio #133 categories
v 0.4.2

### CODE REVIEW

File: l10n/bundle.l10n.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "Do you really want to permanently delete {0}:{1} from the configuration?": "Do you really want to permanently delete {0}:{1} from the configuration?",
   "Cannot show a {0} entity in a {1} Perpective.": "Cannot show a {0} entity in a {1} Perpective.",
-  "Enter a name for a new {0} entity.": "Enter a name for a new {0} entity.",
-  "new {0} name": "new {0} name",
+  "Enter a identifier for a new {0} element.": "Enter a identifier for a new {0} element.",
```
Comment (FabienLelaquais) [inline l10n/bundle.l10n.json:4]: -> an indentifier

File: src/editors/ConfigEditor.ts [hunk 2]
```diff
@@ -409,8 +409,8 @@ export class ConfigEditorProvider implements CustomTextEditorProvider {
         return pv;
       }, nodeType + "_1");
     const newName = await window.showInputBox({
-      prompt: l10n.t("Enter a name for a new {0} entity.", nodeType),
-      title: l10n.t("new {0} name", nodeType),
+      prompt: l10n.t("Enter a identifier for a new {0} element.", nodeType),
```
Comment (FabienLelaquais) [inline src/editors/ConfigEditor.ts]: an indentifier

**Review by FabienLelaquais — APPROVED** [review]
Minor typos.
Well done!

File: package.json [hunk 3]
```diff
@@ -17,8 +17,12 @@
     "vscode": "^1.73.1"
   },
   "categories": [
+    "Data Science",
```
Comment (dinhlongviolin1) [inline package.json:20]: Thanks!

**Review by dinhlongviolin1 — APPROVED** [review]

File: l10n/bundle.l10n.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply l10n/bundle.l10n.json:4]: generated

**Pull request merged**